app/src/main/AndroidManifest.xml

```diff
@@ -11,12 +11,13 @@
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
     android:theme="@style/OppiaTheme">
-    <activity android:name="org.oppia.app.topic.conceptcard.testing.ConceptCardFragmentTestActivity" />
-    <activity android:name="org.oppia.app.player.state.testing.StateFragmentTestActivity" />
-    <activity android:name=".player.exploration.ExplorationActivity" />
-    <activity android:name=".topic.questionplayer.QuestionPlayerActivity" />
-    <activity android:name=".topic.TopicActivity" />
-    <activity android:name=".player.audio.testing.AudioFragmentTestActivity" />
+    <activity android:name="org.oppia.app.topic.conceptcard.testing.ConceptCardFragmentTestActivity"/>
```

Comment:
Suggest keeping the formatting consistent to avoid changing existing lines.

Reply:
Kept it as before.